src/Features/CSharp/Portable/AssignOutParameters/AssignOutParametersAboveReturnCodeFixProvider.cs

```diff
@@ -42,6 +42,14 @@ protected override void TryRegisterFix(CodeFixContext context, Document document
         {
             var generator = editor.Generator;
 
+            if (exprOrStatement is LocalFunctionStatementSyntax { ExpressionBody: { } localFunctionExpressionBody })
```

Comment:
Tagging @CyrusNajmabadi for the product change, as I believe he added these code fixes.

Reply:
The results here were totally wrong. The code fix made a change in an unrelated method, with two impacts:

1. The original diagnostic was still reported in the same place, with a code fix offered
2. The iterative application of code fixes was an infinite loop

There was no way to port the test of local functions without this change.

Reply:
yikes!